sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_pipeline_component_builder.py

```diff
@@ -145,6 +145,7 @@ def __init__(
         default_datastore=None,
         tags=None,
         source_path=None,
+        non_pipeline_parameters=None
```

Comment:
I'm thinking we should call this non_pipeline_inputs in v2. 
Although we may keep support the old name to ease our migration

Reply:
support non_pipeline_inputes and non_pipeline_parameters